external/date.js

```diff
@@ -228,7 +228,7 @@ $.extend( _Date.prototype, {
 	equal: function( other ) {
```

Comment:
Right, seems unused within calendar and datepicker. Perhaps we should reconsider this when we decide what happens with this file at all.

@rxaviers What's your opinion on that?